### PR TITLE
Igal/enable cors

### PR DESCRIPTION
**Pull request by tsorya**
verified manually. Not sure how we can test not manually.

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -170,6 +171,13 @@ func main() {
 		ManagedDomainsAPI:   domainHandler,
 		InnerMiddleware:     metrics.WithMatchedRoute(log.WithField("pkg", "matched-h"), prometheusRegistry),
 	})
+	if Options.Auth.AllowedDomains != "" {
+		allowedDomains := strings.Split(strings.ReplaceAll(Options.Auth.AllowedDomains, " ", ""), ",")
```
Comment (ErezAlster) [inline cmd/main.go:175]: why do we need remove spaces? :)

Comment (tsorya) [reply cmd/main.go:175]: cause u can set "openshift.com, *"
and "*" will not work as u expect, u need to trim spaces.
Already felt on this

**Pull request merged**